python/ray/ml/integrations/torch.py

```diff
@@ -0,0 +1,188 @@
+from typing import Optional, Union, Dict, List
+
+import numpy as np
+import pandas as pd
+import torch
+
+from ray.ml.predictor import Predictor, DataBatchType
+from ray.ml.constants import PREPROCESSOR_KEY
+from ray.ml.constants import MODEL_KEY
+from ray.ml.preprocessor import Preprocessor
+from ray.ml.checkpoint import Checkpoint
+
+
+def convert_pandas_to_torch_tensor(
```

Comment:
Make a private function, and move to end of file.

Reply:
This function is also being used by Datasets.

Reply:
Let's split the file up into `torch_utils.py` `torch_predictor.py` then. Having multiple classes of unrelated types mixed into one "torch" file makes it harder to find the predictor class.

Reply:
Yep good idea, updated!